src/modules/vendor/components/programs/cards/templates/FillTheGaps.vue

```diff
@@ -1,16 +1,28 @@
 <template>
   <div class="container">
-    <ni-input class="q-mb-lg" caption="Texte" v-model="card.gappedText" required-field
-      @blur="updateCard('gappedText')" :error="v$.card.gappedText.$error" type="textarea" @focus="saveTmp('gappedText')"
-      :error-message="gappedTextTagCodeErrorMsg" :disable="disableEdition" />
+    <ni-banner icon-color="copper-500" class="bg-copper-grey-100 text-copper-grey-800" icon="info_outline">
+      <template #message>
+        Remplacer chaque bonne réponse par une balise &lt;trou&gt; et cocher les bonnes réponses dans la liste des mots
+      </template>
+    </ni-banner>
+    <div class="q-mb-lg ">
+      <ni-input caption="Texte" v-model="card.gappedText" required-field @focus="saveTmp('gappedText')"
+        @blur="updateCard('gappedText')" :error="v$.card.gappedText.$error" type="textarea" :disable="disableEdition"
+        :error-message="gappedTextTagCodeErrorMsg" />
+      <p :class="`text-italic text-14 text-${filledText.color}`" v-if="filledText.text">
+        Texte final : {{ filledText.text }}
+      </p>
+    </div>
     <q-checkbox v-model="card.canSwitchAnswers" @update:model-value="updateCard('canSwitchAnswers')"
-      label="Réponses interchangeables" class="q-mb-lg" dense :disable="disableEdition" />
+      label="Réponses interchangeables" class="q-mb-lg switch-answers" dense :disable="disableEdition" />
     <div class="row gutter-profile-x">
-      <div v-for="(answer, i) in card.falsyGapAnswers" :key="i" class="col-md-6 col-xs-12 answers">
-        <ni-input class="input" v-model="card.falsyGapAnswers[i].text" :disable="disableEdition"
-          @blur="updateTextAnswer(i)" :caption="`Mot ${i + 1}`" @focus="saveTmp(`falsyGapAnswers[${i}].text`)"
-          :error="getError('falsyGapAnswers', i)" :error-message="falsyGapAnswersErrorMsg(i)"
+      <div v-for="(_, i) in card.gapAnswers" :key="i" class="answers">
+        <ni-input :caption="`Mot ${i + 1}`" v-model="card.gapAnswers[i].text" class="input"
```

Comment:
C'est pas plus simple de travailler avec `(answer, i) in card.gapAnswers` et de passer `answer.text` dans le v-model de l'input ?

Reply:
done